opencart1.5.x/upload/catalog/controller/payment/xendit.php

```diff
@@ -34,10 +36,17 @@ public function process_payment() {
 
         $store_name = $this->config->get('config_name');
         $amount = (int)$order['total'];
+        $invoice_hash = $_POST['invoice_hash'];
 
-        if ($amount < self::MINIMUM_AMOUNT) {
+        if ($invoice_hash != 'credit_card' && $amount < self::MINIMUM_AMOUNT) {
             $json['error'] = 'The minimum amount for using this payment is IDR ' . self::MINIMUM_AMOUNT . '. Please put more item(s) to reach the minimum amount. Code: 100001';
+        } else if ($invoice_hash == 'credit_card' && $amount < self::MINIMUM_AMOUNT_CC) {
+            $json['error'] = 'The minimum amount for using this payment is IDR 5,000. Please put more item(s) to reach the minimum amount. Code: 100001';
```

Comment:
just being consistent with line 42
what do you think of this?
```
'The minimum amount for using this payment is IDR ' . self::MINIMUM_AMOUNT_CC . '.
```

Reply:
In here I just want to give thousand separator actually especially for 200,000,000. It hard to read 200000000 that why I made it different.

Reply:
I see ok, it seems just formatting
is there any standard formatting in PHP?
should we use this? https://www.php.net/manual/en/function.number-format.php

Reply:
let me try then

Reply:
you can use `number_format(self::MINIMUM_AMOUNT_CC)`